src/pages/ADN/style.less

```diff
@@ -0,0 +1,45 @@
+@import './../pages';
```

Comment:
The styling for 'add to mal' is still broken. But everything else looks good

![image](https://user-images.githubusercontent.com/12599334/214126281-faad0443-47fe-4ae0-8b8c-9cb23706d12d.png)